fern/docs.yml

```diff
@@ -519,6 +519,9 @@ navigation:
           - page: Generate Webhook Reference
             icon: fa-regular fa-webhook
             path: ./pages/docs/api-references/generate-webhook-ref.mdx
+          - page: OpenAPI v2 Parser
```

Comment:
I don't think you want to call it this publicly. V2 implies Swagger whereas V3 implies OpenAPI in this context.

Page title could be:
- Parsing the OpenAPI Specification
- OpenAPI Parser